package/libs/readline/Makefile

```
@@ -70,6 +70,8 @@ endef
 define Package/libreadline/install
 	$(INSTALL_DIR) $(1)/usr/lib
 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/lib{history,readline}.so.* $(1)/usr/lib/
```

```suggestion
	$(CP) $(PKG_INSTALL_DIR)/usr/lib/lib{history,readline}.so* $(1)/usr/lib/
```
I would prefer this solution.

---

Yes, done

---

`find $(PKG_BUILD_DIR) -name \*.so* -exec cp {} $(1)/usr/lib/ \;` has worked wonders for me exactly for preserving symlinks as well for libraries.